bin/node/runtime/src/lib.rs

```diff
@@ -2049,10 +2050,18 @@ impl_runtime_apis! {
 		}
 	}
 
-	impl pallet_staking_runtime_api::StakingApi<Block, Balance> for Runtime {
+	impl pallet_staking_runtime_api::StakingApi<Block, Balance, AccountId> for Runtime {
 		fn nominations_quota(balance: Balance) -> u32 {
 			Staking::api_nominations_quota(balance)
 		}
+
+		fn eras_stakers(era: sp_staking::EraIndex, account: AccountId) -> pallet_staking::Exposure<AccountId, Balance> {
+			Staking::api_eras_stakers(era, account)
+		}
```

Comment:
now's your last time to ask yourself: is the (legacy) name `eras_stakers` actually good? shoudl we not use the oppurtunity and rename it to something better? :D

Reply:
I don't have a strong opinion here. 

On one hand, since this is meant for backward compatibility and all similar storage items are named similarly (eras_stakers, eras_stakers_paged, eras_stakers_overview ...), it makes sense to keep a similar name. 

We could call it `eras_exposure` as well but then we are exposing a new term (exposure) outside staking world which may be just an internal concept. `Eras Stakers` might not be such a bad public name (staker info for the era).

Do you have a name suggestion?